content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -448,16 +448,19 @@ A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; the
 
 #### DNS
 
-A ServiceAccount for `kube-dns` is created in the `kube-system` namespace.
+Please note that:
+1. The CoreDNS service is named `kube-dns`. This is done to prevent any interruption
```

Comment:
In Hugo/Blackfriday, you need a space between the intro text ("Please note that:") and any sort of list. Also, remove "Please" bc it's information the user needs straight up. "Please" gets in the way.

Also, please make this into a bulleted list. It's not a series of steps in a procedure that must be performed in order. (Issue throughout the file, but I'm not asking you to fix it everywhere ;)).

(A usability note: it makes my head hurt to think about a service for CoreDNS being named the thing it's replacing as the default. This seems like confusing UX now, and possibly worse down the road. Are there any plans to fix up in a subsequent release?)

Reply:
@rajansandeep 

> (A usability note: it makes my head hurt to think about a service for CoreDNS being named the thing it's replacing as the default. This seems like confusing UX now, and possibly worse down the road. Are there any plans to fix up in a subsequent release?)

do you know the roadmap for the above?

Reply:
You can't change that without potential down time. name and ClusterIP are immutable on Services, and you can't use the same ClusterIP for two services. So, they only way to fix it is to actually lose DNS service for a very short window. we decided in sig-network that the service name refers to the service, regardless of the specific implementation that is providing the service.

Reply:
@neolit123 this will remain until kube-dns and CoreDNS are both available to install. This can change once kube-dns is deprecated eventually, but for the time being there is no plan to change this. 

@johnbelamaric

Reply:
@johnbelamaric @rajansandeep thanks for the details.
@Bradamant3 we have to keep this note. i will now fix the rest of the text formatting issues.